lib/src/model/uuid.dart

```diff
@@ -63,13 +64,11 @@ class Uuid {
   }
 
   @override
-  int get hashCode =>
-      data.fold(17, (hash, octet) => 37 * hash + octet.hashCode);
+  int get hashCode => data.fold(17, (hash, octet) => 37 * hash + octet.hashCode);
 
   @override
   bool operator ==(dynamic other) =>
-      other.runtimeType == runtimeType &&
-      const DeepCollectionEquality().equals(other.data, data);
+      other.runtimeType == runtimeType && const DeepCollectionEquality().equals(other.data, data);
```

Comment:
Do these line really fit into 70 characters column?

I'd recommend to run a formatter on all the files, because if the format is not right, we'll get a penalty in the rating on pub.dev.

Reply:
we do run a formatter in the CI but also locally it does not change